monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md

```diff
@@ -0,0 +1,57 @@
+# Быстрое создание кошелька Monero
+*04/22/19*  
+Almutasim  
+_**Создание кошелька является одним из самых важных решений, которые приходится принимать пользователю.**_
+
+
+Программное обеспечение кошелька Monero представляет собой программу для хранения, отправки и получения Monero. Оно обеспечивает полный контроль над вашими Monero, гарантируя анонимность и отсутствие каких-либо посредников. Никто не сможет узнать всех подробностей ваших транзакций, равно как и не сможет изменить их, если только вы не поделитесь своими ключами просмотра для проведения аудита. Эта особенность является сильной стороной и сущностью Monero.
+
+Для того, чтобы использовать эту возможность, вам будет необходимо создать кошелёк. Первым критическим аспектом любого кошелька является его надёжность. В этой статье упоминаются только те кошельки, которые рекомендуются сообществом (и веб-сайтом [getmonero.org](https://getmonero.org/)). Затем, вам будет необходимо выбрать тот кошелёк, который будет работать с вашим вычислительным устройством: iPhone, мобильным телефоном с операционной системой Android или же с портативным или стационарным компьютером. Наконец, вы выберите тот кошелёк, который будет соответствовать вашим требованиям к уровню анонимности. И эта статья поможет вам сделать выбор.
```

Comment:
For the link we can redirect to the russian getmonero site:

https://getmonero.org/ru/

Reply:
fix